### PR TITLE
Adding instructions for creating a multi-node cluster in Timescale Forge

### PR DESCRIPTION
**Pull request by ryanbooz**
Adding instructions for a user to install a DIY multi-node cluster on Timescale Forge.

### CODE REVIEW

File: getting-started/forge-multi-node.md [hunk 1]
```diff
@@ -0,0 +1,256 @@
...
+
+## Step 1: Create Services for Access and Data node Services
+
+First, you need to create new Services within your Forge account. As mentioned 
+earlier, you should create _at least_ three Services to set up a multi-node cluster: 
+one access node and two data nodes. 
+
+Because there is currently no way to distinguish between the access node and data 
+nodes within the Timescale Forge console, **we strongly recommend that you include 
+“AN” and “DN” in the names of each service, respectively (eg. “AN-mycluster”,
```
Comment (niksajakovljevic) [inline getting-started/forge-multi-node.md:43]: imho more natural naming is `mycluster-an` or `mycluster-dn`

File: getting-started/forge-multi-node.md [hunk 2]
```diff
@@ -0,0 +1,256 @@
...
+
+## Step 2: Upgrade the Services to v2.0
+
+Currently, any newly created Service in Timescale Forge still uses TimescaleDB 
+1.7.4 by default. This will be the case until we release TimescaleDB 2.0 for 
+production (sometime in late December 2020). Therefore,  to enable multi-node 
+functionality, you need to manually upgrade each TimescaleDB Service to the newest 
+ TimescaleDB 2.0 version available, currently Release Candidate 3 (RC3).
+
+To do this, we recommend using `psql` to connect to all Services created in Step 1
```
Comment (niksajakovljevic) [inline getting-started/forge-multi-node.md:65]: maybe bold :)

File: getting-started/forge-multi-node.md [hunk 3]
```diff
@@ -0,0 +1,256 @@
...
+-------------+-----------+------------+---------------------------------------
+ plpgsql     | 1.0       | pg_catalog | PL/pgSQL procedural language
+ timescaledb | 2.0.0-rc3 | public     | Enables scalable inserts and complex queries for time-series data
+```
+
+## Step 3: Add Data Nodes to the cluster
+
+Once you’ve created your new Services and upgraded TimescaleDB, you’ll enable 
+communication between the access node and all data nodes. The currently supported 
+method for enabling communication between nodes is through **user mapping authentication**.
```
Comment (niksajakovljevic) [inline getting-started/forge-multi-node.md:105]: maybe a link pointing to PG docs: https://www.postgresql.org/docs/current/sql-createusermapping.html

File: getting-started/forge-multi-node.md [hunk 4]
```diff
@@ -0,0 +1,256 @@
...
+```SQL
+CREATE ROLE mn_user1 LOGIN PASSWORD 'password';
+GRANT ALL PRIVILEGES ON ALL TABLES IN SCHEMA public TO mn_user1;
+```
+
+While connected to the access node we can use a specially designed function to 
+execute SQL commands against data nodes:
+
+```SQL
+CALL distributed_exec(query => 'CREATE ROLE mn_user1 LOGIN PASSWORD $$password$$', node_list => '{dn1}');
```
Comment (niksajakovljevic) [inline getting-started/forge-multi-node.md:216]: Maybe extend `node_list` to `{dn1,dn2}` and mention in the sentence above that we assume that you've previously added `dn1` and `dn2` as data nodes.

Comment (niksajakovljevic) [reply getting-started/forge-multi-node.md:216]: Same  list would go for the query below of course

**Review by niksajakovljevic — APPROVED** [review]

File: getting-started/forge-multi-node.md (hunk 1 above)
Comment (ryanbooz) [reply getting-started/forge-multi-node.md:43]: "... there are two hard things in programming..." 😂. I don't feel strongly either way, the only advantage of this for the how-to is that it's easier for me to spot up front in the SQL when 'dn1' is using 'DN1-mycluster'.

File: getting-started/forge-multi-node.md (hunk 3 above)
Comment (ryanbooz) [reply getting-started/forge-multi-node.md:105]: Added below on line 127 as part of the larger discussion. Can change later if a problem.

**Review by solugebefola — COMMENTED** [review]
Mostly suggestions, but a few grammatical/spelling things that need to be fixed.

**Pull request merged**